.changie.yaml

```
@@ -0,0 +1,18 @@
+changesDir: .changes
+unreleasedDir: unreleased
+changelogPath: CHANGELOG.md
+versionExt: md
+versionFormat: '## {{.Version}} ({{.Time.Format "January 02, 2006"}})'
+kindFormat: '### {{.Kind}}'
```

To match existing category headers, which do not use Markdown headings:

```suggestion
kindFormat: '{{.Kind}}:'
```